happi/backends/mongo_db.py

```diff
@@ -64,8 +64,8 @@ def __init__(self, host=None, user=None,
                                 'port specified at startup')
```

Comment:
Unrelated to this line: you missed updates in happi/backends/core.py that implicitly set the backend interface

Reply:
Good catch - 0616a63

I expect this won't be the last you find